jwst/assign_wcs/nircam.py

```diff
@@ -48,11 +48,15 @@ def imaging_distortion(input_model, reference_files):
 
 
 exp_type2transform = {'nrc_image': imaging,
-                      'nrc_slitless': not_implemented_mode, #WFSS mode
-                      'nrc_tacq': not_implemented_mode,#       ?? distortion
-                      'nrc_coron': not_implemented_mode,#    ?? distortion
-                      'nrc_focus': not_implemented_mode,#       ?? distortion
-                      'nrc_tss': not_implemented_mode,# custom SOSS like mode TBC
-                      'nrc_tsi': not_implemented_mode,# custom soss like mode (TBC
-                      'nrc_led': not_implemented_mode,#    ?? WFSS mode
+                      'nrc_grism': not_implemented_mode,
+                      'nrc_tacq': imaging,
+                      'nrc_taconfirm': imaging,
+                      'nrc_coron': not_implemented_mode,
+                      'nrc_focus': imaging,
+                      'nrc_tsimage': imaging,
+                      'nrc_tsgrism': not_implemented_mode,
+                      'nrc_led': not_implemented_mode,
+                      'nrc_dark': not_implemented_mode,
+                      'nrc_flat': not_implemented_mode,
+                      'nrc_wfsc': not_implemented_mode,
```

Comment:
Confirmed with Daryl Swade that NRC_WFSC is NOT a legal exp_type value. So it should at least be removed here and would be good to remove from core schema as well.